corehq/motech/dhis2/templates/dhis2/partials/dhis2_dataset_map_templates.html

```diff
@@ -102,13 +102,18 @@
 
         <div class="form-group">
             <label class="control-label col-sm-3 col-md-2 requiredField">
-                {% trans "UCR Report ID" %}<span class="asteriskField">*</span>
+                {% trans "User Configurable Report" %}<span class="asteriskField">*</span>
             </label>
             <div class="controls col-sm-9 col-md-8 col-lg-6">
-                <input class="textinput textInput form-control"
-                       required=""
-                       type="text"
-                       data-bind="value: ucrId" />
+                <select class="textinput textInput form-control"
+                        required=""
+                        data-bind="value: ucrId">
+                  {% for ucr in ucrs %}
+                    <option value="{{ ucr.get_id }}">{{ ucr.title }}</option>
+                  {% empty %}
+                    <option value="">No UCRs defined</option>
```

Comment:
Does this render well?  Seems like it'd be a select box with one option, "No UCRs defined", but that option would be a perfectly fine, selectable option.

Reply:
Yes, that is exactly what it is :upside_down_face: Maybe I should change this to some help, and a link to define UCRs.

Reply:
165193e51ab